Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -123,18 +123,16 @@ class ActivityStreamTest: BaseTestCase {
         }
         waitUntilPageLoad()
 
-        app.buttons[AccessibilityIdentifiers.Toolbar.settingsMenuButton].tap()
-        app.otherElements[ImageIdentifiers.addShortcut].tap()
         // Workaround to have visited website in top sites
         navigator.performAction(Action.AcceptRemovingAllTabs)
         navigator.performAction(Action.CloseURLBarOpen)
         navigator.performAction(Action.OpenNewTabFromTabTray)
 
-        waitForExistence(app.collectionViews.cells.staticTexts[newTopSite["topSiteLabel"]!], timeout: TIMEOUT)
-        XCTAssertTrue(app.collectionViews.cells.staticTexts[newTopSite["topSiteLabel"]!].exists)
+        waitForExistence(app.collectionViews.cells.staticTexts[newTopSite["bookmarkLabel"]!], timeout: TIMEOUT)
```

Comment:
I think these are the only changes needed.... but if there is not need to add it as a shourtcut to be sure it appears here, I guess it is fine to remove the two lines above.